README.md

```diff
@@ -9,6 +9,8 @@ it on [Google Play](https://play.google.com/store/apps/details?id=org.wordpress.
 ## Build Instructions ##
 
 1. Make sure you've installed [Android Studio](https://developer.android.com/studio/index.html).
+1. `brew install imagemagick@6` to install the correct ImageMagick formula.
```

Comment:
Should we note that you need homebrew installed? 🤔

Reply:
Ooh, good catch!  I’ll add that.

Reply:
Since this isn't needed to build the app, should this instruction be somewhere else?

It is needed to run `bundle install` and run the fastlane tasks but that isn't typically needed.

Reply:
Good question.  Is there a better place we could capture this knowledge?

Reply:
You could add it to the "Notes" section below this. Alternatively, you could add a check to the `Gemfile` to prompt the user to install it is its not present.

Reply:
> add a check to the Gemfile to prompt the user to install it is its not present.
Oh cool!  I didn’t know you could do that!  :)  I like that the best, since it keeps the notes cleaner and not everyone may need to run the command.  WDYT?  Would it be appropriate to open a new issue to track this?

Reply:
Yes, opening an issue would be great!